### ISSUE TITLE
Tox and Docs

### ISSUE THREAD

**Issue by garrettwrong** [ComputationalCryoEM/ASPIRE-Python#828]
Running tox after generating the docs results in the following.

Probably we can tell tox to ignore the doc build artifacts.

```
tox -e check
check: install_deps> python -I -m pip install black check-manifest flake8 flake8-bugbear flake8-colors flake8-print isort twine
check: commands[0]> flake8 .
./docs/build/html/_downloads/7ae0b9a41836bdfffb6e66391d0a0718/configuration.py:91:1: T201 print found.
./docs/build/html/_downloads/7ae0b9a41836bdfffb6e66391d0a0718/configuration.py:96:1: T201 print found.
./docs/build/html/_downloads/7ae0b9a41836bdfffb6e66391d0a0718/configuration.py:98:1: T201 print found.
./docs/build/html/_downloads/7ae0b9a41836bdfffb6e66391d0a0718/configuration.py:103:1: T201 print found.
./docs/build/html/_downloads/7ae0b9a41836bdfffb6e66391d0a0718/configuration.py:115:1: E402 module level import not at top of file
./docs/build/html/_downloads/7ae0b9a41836bdfffb6e66391d0a0718/configuration.py:117:1: T201 print found.
./docs/build/html/_downloads/7ae0b9a41836bdfffb6e66391d0a0718/configuration.py:121:1: T201 print found.
./docs/source/auto_tutorials/configuration.py:91:1: T201 print found.
./docs/source/auto_tutorials/configuration.py:96:1: T201 print found.
./docs/source/auto_tutorials/configuration.py:98:1: T201 print found.
./docs/source/auto_tutorials/configuration.py:103:1: T201 print found.
./docs/source/auto_tutorials/configuration.py:115:1: E402 module level import not at top of file
./docs/source/auto_tutorials/configuration.py:117:1: T201 print found.
./docs/source/auto_tutorials/configuration.py:121:1: T201 print found.
check: exit 1 (2.31 seconds) /Users/gbwright/work/ASPIRE-Python.conda_opts> flake8 . pid=54341
  check: FAIL code 1 (9.76=setup[7.45]+cmd[2.31] seconds)
  evaluation failed :( (10.14 seconds)
  ```

**Comment by j-c-c** [comment]
One solution could be to figure out why `logger` messages are not being captured in the gallery. I believe they should be. If we fix that, then we can replace these print statements.

Here is a related issue (which appears to be resolved) in the sphinx github: https://github.com/sphinx-gallery/sphinx-gallery/issues/672

**Comment by garrettwrong** [comment]
IIRC some of the print statements are from code blocks as an example of what a user would do. So, the prints were somewhat intentional; at least like in this code that I added (which I think is where they came from 😇 ):

https://github.com/ComputationalCryoEM/ASPIRE-Python/blob/master/gallery/tutorials/configuration.py

Might be easiest to just add the doc generated folders (`./docs/build/`, ./docs/source/auto_tutorials) to the tox ignores (or equivalent) if we want to keep the ability to use `print()` for galleries.  Just using the logger would also make the complaints go away, but might have some complication like you mention.

I just ran a `git clean` to run tox and moved on, but I don't think we should need to do that.
